samples/precompiled/HttpSyncStart.cs

```diff
@@ -24,11 +24,11 @@ public static class HttpSyncStart
 
             log.Info($"Started orchestration with ID = '{instanceId}'.");
 
-            return await starter.CreateCheckStatusResponse(
+            return await starter.WaitForCompletionOrCreateCheckStatusResponseAsync(
                 req,
-                instanceId,
-                TimeSpan.FromSeconds(2),
-                TimeSpan.FromMilliseconds(50));
```

Comment:
In my previous feedback, I suggested having values that come from the query string, but that feedback was only intended to apply to the sample code. This change pulls the timeout values from the request object in _all_ cases, which was not my intention. I feel that is too limiting for users who either want a different way of specifying these values or don't want the caller to specify them at all.

I would prefer if `timeout` and `retryInterval` remained as explicit parameters. Only the `HttpSyncStart` sample should look at the request query string values.

Reply:
Hello @cgillum ,

Thank you very much for the clarification. I will work on the changes and submit them as soon as possible.

Thank you!